src/server/protocol.ts

```diff
@@ -19,11 +19,8 @@ import {
     TypeAcquisition,
 } from "./_namespaces/ts";
 
-/**
- * Declaration module describing the TypeScript Server protocol
- */

```

Comment:
This JSDoc comment is on the wrong declaration now that we are modules; noticed this while working on the PR.